sprout/appliances/tasks.py

```diff
@@ -429,9 +429,13 @@ def create_appliance_template(self, provider_id, group_id, template_name, source
                 source_template = None
         else:
             source_template = None
-        template = Template(
-            provider=provider, template_group=group, name=new_template_name, date=date,
-            version=template_version, original_name=template_name, parent_template=source_template)
+        template = Template(provider=provider,
+                            template_group=group,
+                            name=new_template_name,
+                            date=template_info.datestamp,
+                            version=template_version,
+                            original_name=template_name, parent_template=source_template,
+                            exists=False)
```

Comment:
Q: what is it for ?

Reply:
Discussed in chat - at this point the template does not exist, and the model defaults to True. exists attribute is updated at the end of prepare_template_finish already.